src/utils/Filter.ts

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+
+export const Filter = (people: Person[], searchParams: URLSearchParams) => {
```

Comment:
```suggestion
export const filterPeople = (people: Person[], searchParams: URLSearchParams) => {
```
name functions in camelCase